### DISCUSSION TITLE
Develco Frient SMSZB-120 Smoke detector Fail to correctly pair

### DISCUSSION THREAD

**Discussion by boelle** [Koenkk/zigbee2mqtt#17322] [Devices]
i have one of these that runs fine... but the 2nd one fails

i got an replacment but still  it will not configure correct

2023-04-13 15:45:05Failed to configure '0x0015bc0031017ea6', attempt 3 (Error: Bind 0x0015bc0031017ea6/35 genBasic from '0x00212effff064a2e/1' failed (Error: status: 140) at DeconzAdapter.bind (/app/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:768:19) at Endpoint.bind (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:567:13) at Object.bind (/app/node_modules/zigbee-herdsman-converters/lib/reporting.js:40:9) at Object.configure (/app/node_modules/zigbee-herdsman-converters/devices/develco.js:442:13) at Configure.configure (/app/lib/extension/configure.ts:117:13))

![image](https://user-images.githubusercontent.com/3832239/231780371-5a8fa56c-651d-43f1-9c86-90e078d2ae86.png)

the working one:

![image](https://user-images.githubusercontent.com/3832239/231780522-08824a77-e669-4d85-957c-5053b2462d5e.png)

what could be wrong here? i allready removed it and did the 15 sec reset and still the same

i also sit 2 meters from the raspbee2 controller so distance is not the issue... the working detector is another 4 meters away
https://github.com/Koenkk/zigbee2mqtt/discussions/14479 suggest to move closer... 

i run version 1.30.3-1

where should i look first to further figure things out?

**Comment by boelle** [comment]
20 cm fron the controller and another 15 sec reset did not change anything... a reconfigure also without results

**Reply by boelle** [reply]
EDIT..... that was too close, now i dont even get battery status

**Reply by boelle** [reply]
https://community.home-assistant.io/t/strange-device-pairing-with-zigbee2mqtt/376945/3?u=boelle

the last post got me these kind of errors... red box in upper right

![image](https://user-images.githubusercontent.com/3832239/231800187-d34681c0-44e5-40d3-bfa8-1842aa186df8.png)

**Comment by boelle** [comment]
![image](https://user-images.githubusercontent.com/3832239/231808005-c1c208d5-a856-4d50-a9f5-759380c693d4.png)

i now get 

![image](https://user-images.githubusercontent.com/3832239/231808151-e0bc42e4-245d-4179-a628-905c505d2c21.png)


and i can remote start the sirene...  need to check if the rest is reported or not

**Reply by boelle** [reply]
MQTT publish: topic 'zigbee2mqtt/0x0015bc0031017ea6', payload '{"alarm":null,"battery":100,"battery_low":false,"enrolled":true,"fault":null,"ias_cie_address":"0x00212effff064a2e","linkquality":255,"max_duration":null,"reliability":null,"restore_reports":true,"smoke":false,"supervision_reports":true,"temperature":23.25,"test":false,"update":{"installed_version":-1,"latest_version":-1,"state":null},"update_available":null,"voltage":3100}'


MQTT publish: topic 'zigbee2mqtt/Røgalarm Kontor/Entre', payload '{"alarm":null,"battery":100,"battery_low":false,"enrolled":true,"fault":false,"ias_cie_address":"0x00212effff064a2e","linkquality":255,"max_duration":240,"reliability":"no_fault_detected","restore_reports":true,"smoke":false,"supervision_reports":true,"temperature":22.18,"test":false,"update":{"installed_version":262151,"latest_version":262149,"state":"idle"},"update_available":false,"voltage":3000,"zone_id":23}'


so i still miss: 

fault
max_duration
reliability     - i get that one via reporting... i just noted down the payload differences
update

**Comment by boelle** [comment]
max duration when i add it ro reporting... i get those red box's

![image](https://user-images.githubusercontent.com/3832239/231812372-7252b56a-76ec-4c60-86ff-9f2c9bda2594.png)

**Reply by boelle** [reply]
now trying teh console to see what i can in fact read... will be a very slow process going through every cluster

**Reply by boelle** [reply]
![image](https://user-images.githubusercontent.com/3832239/231826191-43a15854-5d59-4b09-ad3f-15bbdb15f9e5.png)

funny enough i cant add them under reporting

![image](https://user-images.githubusercontent.com/3832239/231827946-d80448ba-e4fc-4c40-ae09-c4a7ab2364a8.png)

battery and battery percent and battery alarm state could be added to reporting

![image](https://user-images.githubusercontent.com/3832239/231830175-60de9ac6-0c8c-4593-8b95-ff36abacca6c.png)

that one could not be added

**Comment by boelle** [comment]
yep, it took some hours...

{
    "battery": 100,
    "battery_low": false,
    "enrolled": true,
    "fault": false,
    "ias_cie_address": "0x00212effff064a2e",
    "linkquality": 255,
    "max_duration": 240,
    "reliability": "no_fault_detected",
    "restore_reports": true,
    "smoke": false,
    "supervision_reports": true,
    "temperature": 19.25,
    "test": false,
    "voltage": 3100,
    "alarm": null,
    "update": {
        "installed_version": -1,
        "latest_version": -1,
        "state": null
    },
    "update_available": null,
    "zone_id": 23
}

so i belive that i dont get alarm and update.... 

hope someone can figure what is going on

**Comment by jamesonuk** [comment]
Lots of reports of this
https://github.com/Koenkk/zigbee2mqtt/issues?q=SMSZB-120
https://github.com/Koenkk/zigbee2mqtt/discussions?discussions_q=SMSZB-120+
and in particular https://github.com/Koenkk/zigbee2mqtt/issues/9235
but no solution.

Appears hit and miss and some devices just work fine and others don't (same reported firmware, next to co-ordinator, full batteries etc.)

**Comment by boelle** [comment]
And strange enough is firmware newer on device than in zigbee2mqtt

man. 17. apr. 2023 10.36 skrev jameson_uk ***@***.***>:

**Comment by joesmith1289** [comment]
My frient smoke detector worked to begin with but now reports very little to home assistant! Has frequent pairing issues, get announcements for OTA updates that dont install. Seems buggy!

**Comment by boelle** [comment]
yep, they are very shitty detectors

price and quality goes hand in hand

you are better off with a normal detector that can be wireless chained to other detectors and then get a zigbee temp sensor
